packages/paste-core/primitives/box/src/PseudoPropStyles.ts

```diff
@@ -6,6 +6,7 @@ export const PseudoPropStyles = {
   _hover: '&:hover',
   _active: '&:active, &[data-active=true]',
   _focus: '&:focus',
+  _focus_hover: '&:focus:hover',
```

Comment:
This change requires a changeset.

Should work, but untested in Paste. Only in codesandbox: https://codesandbox.io/s/bold-dust-10j4or?file=/src/styles.css